common/stacks/src/main/scala/weco/api/stacks/models/display/DisplayResultsList.scala

```diff
@@ -28,9 +28,11 @@ case class DisplayResultsList(
 object DisplayRequest {
   def apply(hold: StacksHold): DisplayRequest = {
     DisplayRequest(
-      item = new DisplayItem(
-        id = hold.canonicalId.map { _.underlying },
-        identifiers = Some(List(DisplayIdentifier(hold.sourceIdentifier)))
+      // TODO: This .get should always be Some here
```

Comment:
I'll refactor this next, but to speed up getting this data into the front-end we could merge this PR, deploy and put the refactor into a new PR.